.github/licenserc.yml

```diff
@@ -27,6 +27,9 @@ header:
       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
       See the License for the specific language governing permissions and
       limitations under the License.
+    
+    pattern: |
+      Copyright The ORAS Authors.
```

Comment:
Adding this pattern seems not right. Instead, can you put a Notary Project license header on top of the original oras license header?

Reply:
Updated.